CanDIG/katsu
# DIG-1244: date intervals part 1 - code only

## What's new
- Update README to include app structure
- Bump katsu version to 4.0.0 to indicate the model change
- Replace CharField with JSONField in models for date
- Update schema with DateInterval object
- Remove date filter (there is a ticket to add it back)
- Update diagnosis_age_count in discovery using date_intervals calculation
- Update tests

### How to test:
- Since there is a model change, need to clean postgres first
- On local: run tox, all tests should pass
- On docker: checkout katsu to this PR and ingest PR https://github.com/CanDIG/candigv2-ingest/pull/56
